### PR TITLE
feat: create dummy blog component

### PR DESCRIPTION
**Pull request by OdunayoOkebunmi**
### What does this PR do?
connects #14

### CODE REVIEW

File: src/screens/App/screens/Main/screens/Blog/components/PostsCard.js [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
 
-        <div className='sm:w-1/5 mt-6 mb-6 xl:mt-0 px-6 md:block'>
-          <p>Tags</p>
-          <hr />
-          <Link to='/blog'>Tag 1</Link>
-          <br />
-          <Link to='/blog'>Tag 2</Link>
-          <br />
-          <Link to='/blog'>Tag 3</Link>
+          <div className='sm:w-1/5 mt-6 mb-6 xl:mt-0 px-6 md:block'>
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Blog/components/PostsCard.js]: Im thinking...should this be its own component too?

File: src/screens/App/screens/Main/screens/Blog/components/FeaturedPost.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import SimpleButton from '@App/shared/components/Buttons/SimpleButton';
+
+const imageStyling = {
+  height: 'auto',
+  background: 'linear-gradient(90deg, #667eea 0%, #764ba2 100%)',
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Blog/components/FeaturedPost.js]: can the background here be the thumbnail of the post

File: src/screens/App/screens/Main/screens/Blog/components/Pagination.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const Pagination = () => (
```
Comment (frknasir) [inline src/screens/App/screens/Main/screens/Blog/components/Pagination.js]: it should take two props (handleNext:func, handlePrev: func)

Comment (OdunayoOkebunmi) [reply src/screens/App/screens/Main/screens/Blog/components/Pagination.js]: Noted

File: public/img/facebook.svg [hunk 4]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (frknasir) [inline public/img/facebook.svg]: hey for icons, i think its best we create them as components as we do not want them to be statically available. check this link out https://github.com/CJIDNG/udeme-pod/tree/develop/src/screens/App/shared/components/Icons

Comment (OdunayoOkebunmi) [reply public/img/facebook.svg]: Noted

**Pull request merged**